builtin/providers/aws/resource_aws_opsworks_custom_layer_test.go

```diff
@@ -23,7 +23,7 @@ func TestAccAWSOpsworksCustomLayer(t *testing.T) {
 		CheckDestroy: testAccCheckAwsOpsworksCustomLayerDestroy,
 		Steps: []resource.TestStep{
 			resource.TestStep{
-				Config: testAccAwsOpsworksCustomLayerConfigCreate(stackName),
+				Config: testAccAwsOpsworksCustomLayerConfigNoVpcCreate(stackName),
```

Comment:
The previous test (testAccAwsOpsworksCustomLayerConfigCreate - currently in master) didn't work for me as it attempts to create a "No VPC" stack, but then tries to create some security groups to assign to the custom layer within this No VPC stack. The creation of these security groups then fails without a default VPC being set.
It therefore made sense to test for the VPC case, and call the stack creation function that also creates a VPC so that we could also test custom security group functionality. That's why I've branched out the testAccAwsOpsworksCustomLayerConfigCreate function into two separate functions - one that creates a VPC stack, and one that creates a non-VPC stack. The non-VPC stack function is exactly the same as before, and this is called as normal for the aws_opsworks_custom_layer_test config. The new VPC stack function is only called from the import test.

I thought that creating these two separate functions was tidiest (and is modelled on the VPC/non-VPC cases from the stack tests). If you'd prefer, we could revert to using the non-VPC test and then set a default VPC for adding the security groups? Alternatively, we could turn off custom security groups in the test and use built-in OpsWorks security groups - a default VPC does not need to be set when these are used.